lib/rules/prefer-template.js

```diff
@@ -147,7 +147,7 @@ module.exports = {
                         return `\\${matched}`;
                     }
                     return matched;
-                })}\``;
+                }).replace(new RegExp(`\\\\${currentNode.raw[0]}`, "g"), currentNode.raw[0])}\``;
```

Comment:
Maybe this is a nitpick, but what do you think about adding a comment above this, explaining that this part is for unescaping quotes?